authors/apps/articles/renderers.py

```diff
@@ -26,3 +26,21 @@ def render(self, data, media_type=None, renderer_context=None):
                 'articles': data,
                 'articlesCount': len(Article.objects.all())
             })
+
+
+class LikeStatusRenderer(JSONRenderer):
+    charset = 'utf-8'
+
+    def render(self, data, media_type=None, renderer_context=None):
```

Comment:
What does this method do?

Reply:
When come one likes an article its the renderer that renders my data, It returns the Like_status_id, the article title and id plus the username of the person choosing to like/dislike an article